ingestion/src/metadata/profiler/processor/models.py

```diff
@@ -30,26 +47,4 @@ class ProfilerDef(BaseModel):
     timeout_seconds: Optional[
         int
     ] = None  # Stop running a query after X seconds and continue
-    metrics: Optional[
-        List[str]
-    ] = None  # names of currently supported Static and Composed metrics
-    # TBD:
-    # time_metrics: List[TimeMetricDef] = None
-    # custom_metrics: List[CustomMetricDef] = None
-    # rule_metrics: ...
-
-    # pylint: disable=no-self-argument
-    @validator("metrics", each_item=True)
```

Comment:
@TeddyCr `@validator` is deprecated in v2. Just saw it and did this change. Added a test just in case, but lmk if you see anything weird

Reply:
there's the alternative annotated fix above in this same file btw